surveys/2024-annual-survey/questions.md

```diff
@@ -414,11 +382,33 @@ Type: select all that apply (optional)
 - Achieving structured concurrency with async code
 - Writing executor-agnostic async code
 - Borrow checker not allowing valid code
+
+- (From here the additional one)
+- Runtime performance
+- Compile times
+- Binary size (of the final compiled artifact)
+- Memory usage (example: how much RAM rustc uses when compiling)
+- Disk space usage (example: the size of target folder)
+- Bugs in the compiler (that is, ICEs a.k.a. internal compiler errors, miscompilations, etc.)
+- Opaque / unclear compiler error messages
+- IDE experience
+- Debugging experience (example: unclear / misleading diagnostics)
+- Documentation (rustdoc, docs.rs)
+- Build system (cargo)
+- Package management (crates.io)
```

Comment:
Using dependencies? Or e.g. auditing dependencies? We can just arbitrarily choose some common complaints :)

Reply:
Heh, this is also a wide topic. I don't know how to change this, what do we really want to point at